app/views/grids/_notes.html.erb

```diff
@@ -21,7 +21,7 @@
   </tr>
 <% if nodes.empty? %>
   <tr>
-    <td>No matching content.</td>
+    <td>Nothing yet on the topic "<b><a href="/tag/<%= tagname %>"><%= tagname %></a></b>" -- be the first to post something!</td>
```

Comment:
Everything is good here. However, you could still improve things by linking `post something` to redirect the user to the page he should visit to post a topic.

Reply:
It could be to... `/post?tags=<%= tagname %>` !! Great idea!

Reply:
Is there any way I can pull request again to improve it as you suggest? @seafr @jywarren thanks for the suggestion!

Reply:
You don't have to do open a PR again.

Just make the changes *while on the `patch-1` branch*, commit and push to the `patch-1` branch on your fork. This PR here will be **automatically** updated.

Reply:
Thanks for the clear explanation @seafr . I employed @jywarren 's suggestion to link it. To be honest, I'm not so sure what I'm doing. I was trying to look for a way to actually test that link and see where it would take me to make sure that it was actually linking to someplace that would make sense, but I didn't have any luck. Thanks for the suggestions anyway!

Reply:
You are welcome. And don't worry, you did as expected and this now is ready to be merged.